.github/workflows/AirspeedVelocity.yml

```
@@ -0,0 +1,78 @@
+name: Benchmark a Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  pull-requests: write
+
+jobs:
+    generate_plots:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v4
+            - uses: julia-actions/setup-julia@v2
+              with:
+                version: "1.10"
+            - uses: julia-actions/cache@v2
+            - name: Extract Package Name from Project.toml
+              id: extract-package-name
+              run: |
+                PACKAGE_NAME=$(grep "^name" Project.toml | sed 's/^name = "\(.*\)"$/\1/')
+                echo "::set-output name=package_name::$PACKAGE_NAME"
```

```suggestion
                echo "package_name=$PACKAGE_NAME" >> $GITHUB_OUTPUT
```
This was giving a warning, see, e.g., https://github.com/JuliaGeometry/MeshIntegrals.jl/actions/runs/11516676483/job/32059817164#step:5:6. This updates the command according to https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.